components/cases/casesSection.tsx

```diff
@@ -25,7 +25,11 @@ const CasesSection = () => {
         summary: removeImageLinksFromMarkdown(item?.attributes?.summary),
         id: item?.id,
       }));
-      setPlatform(transformData);
+
+      const filteredPlatforms = transformData?.filter((platform: IPlatfrom) =>
+        [1, 2, 3].includes(platform.id)
```

Comment:
[1, 2, 3] - what is it?

Reply:
this is the platform id that is displayed on the home page